libs/ui-toolkit/letterhead/src/lib/footer/footer.component.html

```diff
@@ -1,3 +1,6 @@
+<section class="letterhead-footnote">
```

Comment:
Este elemento deve estar presente mesmo se o footnote text for "falsy"? 🤔

Reply:
Da forma que eu fiz sim, para não alterar o espaço ocupado pelo footer de forma mais simples, eu mudei a margem de 120px para 40px e adicionei esse elemento com tamanho fixo de 80px
Acho que dá para tirar ele, mas me parece que a lógica fica desnecessariamente complexa